libs/perun/dialogs/src/lib/delete-entity-dialog/delete-entity-dialog.component.ts

```diff
@@ -25,6 +25,8 @@ export class DeleteEntityDialogComponent {
   disableForce = false;
   @Input()
   loading = false;
+  @Input()
+  anonymize = false;
 
   @Output()
   deleted: EventEmitter<{ deleted: boolean; force: boolean }> = new EventEmitter<{
```

Comment:
Please create an interface for the emitted value. Use this interface in the `anonymizeUserDialog` as well as other dialogs which are "containers" of `deleteEntityDialog`.